pom.xml

```
@@ -121,7 +121,7 @@
     </dependency>
     <dependency>
       <groupId>org.hamcrest</groupId>
-      <artifactId>hamcrest-core</artifactId>
+      <artifactId>hamcrest</artifactId>
```

```suggestion
      <artifactId>hamcrest</artifactId>
      <version>2.2</version>
```
To be removed when updating the parent pom.

---

apologies, does it mean we can get rid of it completely?

---

> apologies, does it mean we can get rid of it completely?

No, just the `<version>2.2</version>` part. Plugin pom versions newer than 4.64 contain `org.hamcrest:hamcrest:2.2`, so this override is no longer needed.